script/code-gen.sh

```diff
@@ -2,30 +2,36 @@
 
 set -e -o pipefail
 
-# shellcheck disable=SC1091 # Cannot check source of this file
-source /go/src/k8s.io/code-generator/kube_codegen.sh
-
-git config --global --add safe.directory "/go/src/${PROJECT_MODULE}"
+export PROJECT_MODULE="github.com/traefik/traefik"
+export MODULE_VERSION="v2"
```

Comment:
If I am not mistaken those variables do not need to be exported, wdyt?

```suggestion
PROJECT_MODULE="github.com/traefik/traefik"
MODULE_VERSION="v2"
```